app/views/fragments/footerJavaScript.scala.html

```diff
@@ -10,3 +11,12 @@
         }
     })(guardian.isModernBrowser);
 </script>
+
+
+
+<script type="text/javascript">
+    var trackerUrl = '@{Config.trackerUrl}';
+    snowplow_name_here("newTracker", "subscriptions", "@{Config.trackerUrl}", {
```

Comment:
Tracker URL was the only reason, so yes I could just expose that and move the rest into a require. The name "snowplow_name_here" is what that horrible script loader with the cloudformation name you commented on brings into the global namespace. Yes I agree its all very funky, just followed the docs here but I think you're right that it needs cleaning up.